script.js

```diff
@@ -20,10 +20,16 @@ function calculateTip() {
     //Display Total
 
     document.getElementById("billTotal").innerHTML = Total;
+    //calculate tip per person
 
-    //calculate price per person
     var resulted = (billAmt * servicequality) / billsplit;
     document.getElementById("result").innerHTML = resulted;
+    resulted = resulted.toFixed(2);
+
+    //total  amount per person
+    var completeTotal = Total / billsplit;
```

Comment:
Think about better names for your variables. `completeTotal` is confusing for anyone not familiar with the exact code logic. That includes you in 3 months.